frontend/package.json

```diff
@@ -129,5 +130,12 @@
    },
    "msw": {
       "workerDirectory": "public"
+   },
+   "pnpm": {
+      "peerDependencyRules": {
+         "allowedVersions": {
+            "graphql": "15"
+         }
+      }
```

Comment:
I remember rush complains about unmet peer deps, as part of QA we should check that this doesn't break updating the submodule in our php app

Reply:
Fair... but this pull *reduces* the number of unmet peer deps, not increases.

Reply:
Oops I didn't read the docs, I thought this config was doing something different. Seems like there's no concern here.

> Unmet peer dependency warnings will not be printed for peer dependencies of the specified range.

https://pnpm.io/package_json#pnpmpeerdependencyrulesallowedversions